src/Analysis/Ast/Impl/Types/Definitions/IArgumentSet.cs

```diff
@@ -43,11 +43,16 @@ public interface IArgument {
         /// Argument annotation type, if any.
         /// </summary>
         IPythonType Type { get; }
-        
+
         /// <summary>
         /// Parameter location in the AST.
         /// </summary>
         Node Location { get; }
+
+        /// <summary>
+        /// Returns true if this is a default argument.
```

Comment:
`Value of the argument is default` I guess?

Reply:
ok